[FIX][14.0-mig-l10n_br_account] l10n_br_account: hack PORCO para eliminar linhas de movimento vazias

marcelsavegnago opened the pull request:
hack PORCO para eliminar linhas de movimento vazias

renatonlima committed:
[REF] remove document_section field in l10n_br_account

renatonlima committed:
[REF] Call _update_taxes to update taxes fields

gabrielcardoso21 committed:
[ADD] Reference to invoices

renatonlima committed:
[ADD] Fiscal Document Invoice views

renatonlima committed:
[FIX] account move line name

renatonlima committed:
[REF] Fiscal invoice actions

renatonlima committed:
[ADD] Payments page in Customer and Vendor Invoices

renatonlima committed:
[FIX] view actions

renatonlima committed:
[FIX] Fiscal invoice action

renatonlima committed:
Added related field in account invoice

renatonlima committed:
[FIX] account invoice methods

renatonlima committed:
Added fiscal_operation_type default value

renatonlima committed:
[REF] fiscal invoice view fields attributes

renatonlima committed:
Update Account and Fiscal Invoice form view

renatonlima committed:
[FIX] state in fiscal invoice view

renatonlima committed:
[REF] l10n_br_account using computed fields from mixin

renatonlima committed:
Update renamed fields in objects, fields and methods

renatonlima committed:
[ADD] l10n_br_account: migration file

renatonlima committed:
[ADD] l10n_br_stock_account: migration file

renatonlima committed:
[REF] account invoice validate and cancel methods

renatonlima committed:
Update fiscal invoice view

renatonlima committed:
[ADD] fiscal wizard until events is not refactoring

renatonlima committed:
Fiscal invoice buttons and methods

marcelsavegnago committed:
[FIX] l10n_br_account: flake8

renatonlima committed:
[FIX] Account Invoice and Fiscal Document Sync methods and state

renatonlima committed:
[REF] Unlink Fiscal Document Line and Allows Invoice Line with Display Type

renatonlima committed:
Update Fiscal Invoice view buttons domains and attrs

renatonlima committed:
[FIX] finalize_invoice_move_lines to use fiscal document number

renatonlima committed:
[REF] post account move only after fiscal doc is authorized

renatonlima committed:
[FIX] fiscal number copy=False

renatonlima committed:
Unlink fiscal document line from account invoice line

renatonlima committed:
pep8

renatonlima committed:
l10n_br_account: rename costs_value to other_value

renatonlima committed:
[FIX] fiscal document dashboard, button attr, val name

renatonlima committed:
[FIX] not use dummy line in invoice line a true display_type

renatonlima committed:
[FIX] Do not try unlink dummy line

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.3.0.0

gabrielcardoso21 committed:
[ADD] Default values in out invoices

OCA-git-bot committed:
l10n_br_account 12.0.3.1.0

renatonlima committed:
[FIX] amount_total in invoice line method

renatonlima committed:
[FIX] fiscal invoice line compute

renatonlima committed:
[ADD] amount discount in customer invoice and vendor bill

renatonlima committed:
[ADD] discount value in invoice lines

renatonlima committed:
[REF] shadowed methods to reuse in create and write method

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.4.0.0

renatonlima committed:
[FIX] do not remove discount field from view

OCA-git-bot committed:
l10n_br_account 12.0.4.1.0

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.4.2.0

renatonlima committed:
pep8

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.4.3.0

mileo committed:
[FIX] Company on invoice tests

OCA-git-bot committed:
l10n_br_account 12.0.4.4.0

mileo committed:
[FIX] Close #1249

mileo committed:
[FIX] Close #1249

mileo committed:
[IMP] Invoice Cancel Wizard

marcelsavegnago committed:
[FIX] l10n_br_account: flake8

mileo committed:
[FIX] Runbot error

mileo committed:
[REM] Cancel/Invalidation/Correction already on fiscal

mileo committed:
[NEW] Invalidade From Invoice

mileo committed:
[IMP] Invoice Wizards

mileo committed:
[FIX] Flake8

mileo committed:
[NEW] nf-e / nfs-e invoice view

mileo committed:
[REF] Adapt l10n_br_account buttons

mileo committed:
[NEW] Invoice Document Status

mileo committed:
[REF] key -> document_key on l10n_br_account

mileo committed:
[REF] number -> document_number on l10n_br_account

mileo committed:
[REF] Drop fiscal_number [REF] document_number l10n_br_account

mileo committed:
[REF] Protect methods of l10n_br_account

mbcosta committed:
[IMP] Method name_get of incoterms return 'CODE - DESCRIPTION'.

mileo committed:
[FIX] Document Name on invoice finalize_invoice_move_lines

mileo committed:
[FIX] Invoice Refund

oca-travis committed:
[UPD] Update l10n_br_account.pot

renatonlima committed:
[FIX] Get only active account tax

marcelsavegnago committed:
[FIX] l10n_br_account: fix financial_move_line_ids compute method

mileo committed:
[NEW] Copy Invoice Fiscal Tax on refund

mileo committed:
[NEW] Invoice Refund Reference document

mileo committed:
[NEW] Optional Use Origin Taxes on refund

[FIX] Correção pos rebase

marcelsavegnago committed:
[FIX] l10n_br_account: flake8

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.5.0.0

gabrielcardoso21 committed:
[REF] Rename views files

gabrielcardoso21 committed:
[FIX] Flake8

mileo committed:
[REF] Fiscal document object dashboard l10n_br_account

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.5.1.0

rvalyi committed:
remove useless default @api.multi

rvalyi committed:
flake8 B902 'self' used for class method

rvalyi committed:
duplicate def of _get_amount_lines

rvalyi committed:
E741 ambiguous variable name 'l'

rvalyi committed:
file not used

mbcosta committed:
[FIX] In name_get of incoterms return reduced name to avoid distortion on screen.

mbcosta committed:
[IMP] Incoterms should been in shadow_fields to be used in Fiscal Document.

renatonlima committed:
[FIX] remove date shadow field

oca-travis committed:
[UPD] Update l10n_br_account.pot

oca-travis committed:
[UPD] Update l10n_br_account.pot

oca-travis committed:
[UPD] Update l10n_br_account.pot

rvalyi committed:
lint: sed -i 's/assertEquals/assertEqual/g'

mbcosta committed:
[MIG][MOV] Localization of Invoice report.

mbcosta committed:
[MIG] BR fiscal fields in Invoice report.

mileo committed:
[REF] Standard OCA website l10n_br_account

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.6.0.0

rvalyi committed:
m2m column typo fix with migration script

mbcosta committed:
[FIX] Pre commit.

mileo committed:
[FIX] Comments

rvalyi committed:
fixed pylint issues:

* Dangerous use of "replace" from view with priority 0 < 99
* Deprecated @api.one
* Loop control variable 'send_invoice' not used within the loop body

rvalyi committed:
[IMP] l10n_br_account: black, isort, prettier

renatonlima committed:
[FIX] pre-commit regression

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
[UPD] README.rst

mileo committed:
[FIX] Pre commit issues

renatonlima committed:
[REF] adapt hook to link dummy document and invoice

renatonlima committed:
[REF] Adapt invoice and invoice line methods to work with dummy document

renatonlima committed:
[REF] invoice views

renatonlima committed:
[REF] Adapt account move and dummy document

gabrielcardoso21 committed:
[FIX] Fiscal operation type

gabrielcardoso21 committed:
[FIX] Refund fiscal_operation_type

renatonlima committed:
[REF] Add argument to get deductible taxes

renatonlima committed:
[REF] Account tax deductible and withholding fields in compute_all dict

renatonlima committed:
[ADD] Use deductible taxes in fiscal operation

renatonlima committed:
[REF] Create tax withholding invoice

renatonlima committed:
[REF] Create deductible account move line entry

renatonlima committed:
l10n_br_account: pre-commit fixes

renatonlima committed:
[REF] remove @api.multi

rvalyi committed:
bring back chatter in invoice fiscal view

rvalyi committed:
bring back cancel and correction mail.thread messages

ygcarvalh committed:
[ADD] Refund test

ygcarvalh committed:
[FIX] Invoice subsequent operation function call in refund

ygcarvalh committed:
[FIX] pre-commit issues

ygcarvalh committed:
[FIX] flake8

rvalyi committed:
update to res.company#fiscal_dummy_id

renatonlima committed:
l10n_br_account: pre-commit

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.8.0.0

OCA-git-bot committed:
l10n_br_account 12.0.8.1.0

ygcarvalh committed:
[FIX] pre-commit

renatonlima committed:
[FIX] WH invoice due date

renatonlima committed:
[FIX] tax_line_move_line_get

ygcarvalh committed:
[FIX] Journal for refund test

oca-travis committed:
[UPD] Update l10n_br_account.pot

renatonlima committed:
[REF] removed tax included from iml total

rvalyi committed:
[MIG] disable views/imports/hook ->installable

rvalyi committed:
[MIG] sed .env.user.company_id -> .env.company

find l10n_br_account -type f -exec sed -i 's/\.env\.user\.company_id/\.env\.company/g' {} \;

rvalyi committed:
[MIG] trivial fixes + enable files OK again

rvalyi committed:
[MIG] migrate pre_init_hook

rvalyi committed:
[MIG] customer invoice form degraded but loading

rvalyi committed:
[MIG] fiscal_invoice_view degraded but loading

rvalyi committed:
[MIG] actions and menus ~OK; 2 actions commented

rvalyi committed:
[MIG] tests degraded for taxes but mostly passing

rvalyi committed:
[MIG] l10n_br_account: Migration to 14.0

rvalyi committed:
[MIG] sed type -> move_type + manual fixes

find l10n_br_account -type f -exec sed -i 's/"type"/"move_type"/g' {} \;

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.8.1.1

renatonlima committed:
[FIX] action domain to show in and out refunds fiscal invoices

renatonlima committed:
[REF] invoice refund method

renatonlima committed:
[REF] if refund has a origin invoice they are reconcile

oca-travis committed:
[UPD] Update l10n_br_account.pot

rvalyi committed:
[FIX] incorterm_id in _shadowed_fields in l10n_br_delivery fixes #1596

oca-travis committed:
[UPD] Update l10n_br_account.pot

renatonlima committed:
method comment

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.9.0.0

renatonlima committed:
Delete fiscal document related to account invoice

renatonlima committed:
Added invoice line amounts fields

antoniospneto committed:
[bugfix] compute_financial

antoniospneto committed:
fix conflict

oca-travis committed:
[UPD] Update l10n_br_account.pot

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.9.0.1

renatonlima committed:
[REF] removed duplicated code from l10n_br_account

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.10.0.0

oca-travis committed:
[UPD] Update l10n_br_account.pot

antoniospneto committed:
remove update_taxes() call from invoice

antoniospneto committed:
fix estimate_tax field on view

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.11.0.0

oca-travis committed:
[UPD] Update l10n_br_account.pot

rvalyi committed:
[MIG] l10n_br_account: Migration to 13.0

rvalyi committed:
[MIG] sed .invoice.line -> .move.line

find l10n_br_account -type f -exec sed -i 's/\.invoice\.line/\.move\.line/g' {} \;

rvalyi committed:
[MIG] sed account.invoice -> account.move

find l10n_br_account -type f -exec sed -i 's/\.invoice"/\.move"/g' {} \;

rvalyi committed:
[MIG] sed invoice_id -> move_id

find l10n_br_account -type f -exec sed -i 's/invoice_id/move_id/g' {} \;

rvalyi committed:
[MIG] sed invoice_line_id -> line_id

find l10n_br_account -type f -exec sed -i 's/invoice_line_id/line_id/g' {} \;

rvalyi committed:
[MIG] commented refund as it will need migration

rvalyi committed:
[MIG] sed date_invoice -> date

find l10n_br_account -type f -exec sed -i 's/_date_invoice/_PROTECT/g' {} \;
find l10n_br_account -type f -exec sed -i 's/date_invoice/date/g' {} \;
find l10n_br_account -type f -exec sed -i 's/_PROTECT/_date_invoice/g' {} \;

rvalyi committed:
[MIG] number -> name

find l10n_br_account -type f -exec sed -i 's/document_number/PROTECT/g' {} \;
grep -r number l10n_br_account # and manual changes number -> name
find l10n_br_account -type f -exec sed -i 's/PROTECT/document_number/g' {} \;

rvalyi committed:
[IMP] l10n_br_account: black, isort, prettier

rvalyi committed:
[MIG] migrate tests

rvalyi committed:
[MIG] mostly migrated the invoice line view

the line view is now defined inside the invoice form!

rvalyi committed:
[IMP] l10n_br_account: black, isort, prettier

rvalyi committed:
[MIG] commented methods and lines to migrate

rvalyi committed:
[REM] remove dead code

from https://github.com/OCA/l10n-brazil/pull/1479

rvalyi committed:
[MIG] mostly migrated invoice views/edition

oca-travis committed:
[UPD] Update l10n_br_account.pot

OCA-git-bot committed:
l10n_br_account 12.0.11.1.0

antoniospneto committed:
[FIX] fix action_send_email call method

rvalyi committed:
[MIG] avoid Runbot warnings

rvalyi committed:
[MIG] make tests pass

rvalyi committed:
[MIG] impl model_create_multi to avoid

constraint errors of sequential creates (debit = credit)

rvalyi committed:
[MIG] fix invoice edition:

- 1 proper v14 context in invoice_line_ids and line_ids
see also https://github.com/OCA/server-tools/issues/2202
- 2 proper <control> fiscal placeholders to inject fiscal
fields directly in the tree elements.
- 3 adjusted a few optional fields in tree views.

renatonlima committed:
[FIX] typo

renatonlima committed:
[MIGR] fiscal document actions

renatonlima committed:
[WIP] Account move line computes

renatonlima committed:
Return some amount values in fiscal tax compute

renatonlima committed:
[WIP] Account move migration

renatonlima committed:
[REF] replaces line_ids to fiscal_line_ids in l10n_br_account's hook

renatonlima committed:
[REF] rename line_ids to account_line_ids for account.move.line in fiscal.document.line

renatonlima committed:
[REF] fiscal document view in l10n_br_fiscal

marcelsavegnago committed:
[FIX] l10n_br_account: replace dummydoc.line_ids to fiscal_line_ids

marcelsavegnago committed:
[FIX] l10n_br_account: action_invoice_cancel and action_invoice_draft

marcelsavegnago committed:
[FIX] l10n_br_account: hack PORCO para eliminar linhas de movimento vazias ou duplicadas

felipemotter committed:
[FIX] Add company in search account.tax

antoniospneto committed:
[MIG] action_invoice_open -> action_post

antoniospneto committed:
[MIG] comment refund code to fix later

antoniospneto committed:
[FIX] action_document_confirm()

antoniospneto committed:
[MIG] Rename cancel and draft methods

antoniospneto committed:
[MIG] comment tax withholding to mig later

antoniospneto committed:
[MIG] rename draft method

rvalyi commented:
@marcelsavegnago vou comentar sobre isso nesses dias, mas basicamente quando rolou toda aquela treta sobre reinventar o IFRS do Odoo ou não eu tinha validad que o conceito de usar o IFRS do Odoo ia funcionar na 14.0 tb, então tenho um prototipo disso para incluir. Vou comentar em breve.

antoniospneto committed:
[FIX] get icmssn_range of context

marcelsavegnago committed:
[FIX] l10n_br_account: fix copy method

marcelsavegnago committed:
[FIX] l10n_br_account: change @api.model to @api.model_create_multi on move create method

renatonlima committed:
[FIX] repartition factor and sign in compute_all method in account.tax

renatonlima committed:
Add price_subtotal and price_total in account.move.line form

renatonlima committed:
pre-commit

renatonlima committed:
Added debit and credit fields in account.move.line form to call onchange

renatonlima committed:
[MIGR] invoice_line_ids form view

renatonlima committed:
pre-commit

renatonlima committed:
Update python lib erpbrasil.assinatura version 1.4.0

renatonlima committed:
l10n_br_nfse: Update python lib erpbrasil.assinatura version 1.4.0

renatonlima committed:
Account move cleanup old comment methods

renatonlima committed:
[REF] _recompute_payment_terms_lines to update payment term lines ref

OCA-git-bot committed:
l10n_br_nfse 14.0.1.8.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

renatonlima committed:
[REF] API request timeout and check if IBPT Service Unavailable

renatonlima committed:
[FIX] Check IBPT API result is None

renatonlima committed:
[REF] IBPT Tests

marcelsavegnago committed:
[FIX] l10n_br_account: action_cancel to button_cancel

renatonlima committed:
Merge pull request #199 from Escodoo/14.0-mig-l10n_br_account-change-action_cancel-to-button_cancel

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

renatonlima committed:
[MIG] l10n_br_account tests

renatonlima committed:
[FIX] _get_price_total_and_subtotal_model

renatonlima committed:
[FIX] Account.move.line create fiscal document ref

rvalyi commented:
@marcelsavegnago eu prometo destravar isso logo depois da gente fazer o merge do l10n_br_account, eu naõ esqueci.

marcelsavegnago commented:
> @marcelsavegnago eu prometo destravar isso logo depois da gente fazer o merge do l10n_br_account, eu naõ esqueci.

Eu vou dar uma olhada mais a fundo mas acho que isso não tem acontecido mais. Pelo menos já faz pelo menos um mês que nao tenho usado esta PR
